bin/har-to-k6.js

```diff
@@ -66,7 +66,7 @@ function read(file) {
 
 function parse(json) {
   try {
-    return JSON.parse(json)
+    return JSON.parse(json.trim())
```

Comment:
Why do we need to trim the json here? `JSON.parse` already skips leading and trailing whitespace, so this would mean we loop over the string twice. Seems a bit wasteful, especially for very large har files.

Reply:
Sorry @allansson, this was an unrelated change that probably should have been kept out of the PR. I did find myself needing to use it to deal with a particular HAR file generated by Fiddler though. I'll try to locate it. Trimming obviously doesn't do any harm if there's nothing to trim, but you are correct that there may be a performance hit when parsing twice.

Reply:
Here is a Fiddler-generated HAR file (renamed to .txt so I could attach):

[fiddler_test.txt](https://github.com/k6io/har-to-k6/files/6475009/fiddler_test.txt)

Despite there not being any visible whitespace to trim, I receive this exception when I try to parse it:

```
SyntaxError: Unexpected token ﻿ in JSON at position 0
    at JSON.parse (<anonymous>)
    at parse (c:\\git\\har-to-k6\\bin\\har-to-k6.js:70:17)
    at Command.run [as _action] (c:\\git\\har-to-k6\\bin\\har-to-k6.js:46:21)
    at Command._run (c:\\git\\har-to-k6\
node_modules\\caporal\\lib\\command.js:413:40)
    at Program._run (c:\\git\\har-to-k6\
node_modules\\caporal\\lib\\program.js:170:16)
    at Program.parse (c:\\git\\har-to-k6\
node_modules\\caporal\\lib\\program.js:276:17)
    at Object.<anonymous> (c:\\git\\har-to-k6\\bin\\har-to-k6.js:33:4)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
```

It turns out that Fiddler generates HAR files in UTF-8 **with BOM**. Removing the Byte Order Mark allows it to be parsed successfully.

The problem arises from the fact that HAR is not considered JSON as it [has its own spec](https://github.com/ahmadnassri/har-spec/blob/master/versions/1.2.md) where it suggests tools "must support and ignore a BOM", whereas JSON [explicitly disallows it](https://datatracker.ietf.org/doc/html/rfc7159#section-8.1)).

Detecting its presence should be easy enough. Want me to attempt this?

Reply:
There's other problems with Fiddler-generated HAR files, unfortunately.

The next failure happens when attempting to parse `postData` when the object exists but the values are empty strings. The fix here would require an update to `aid.js` which features an `emptyObject` function that currently only checks whether the object has empty `keys` but not whether the `keys` actually contain anything.

I made some changes and have tests working on my local system, but this should probably go into a separate commit at this point. Given that Fiddler-generated HAR files likely never worked, and that nobody has really complained about it, I wonder fi it's even worth fixing it. LMKWYT @allansson